modules/ROOT/pages/microsoft/microsoft-dynamics-365-connector.adoc

```diff
@@ -134,7 +142,7 @@ To build and run this demo project, you need:
 ** dynamics365.clientId - The application ID assigned to your app when you registered it with Azure Active Directory. You can find this in the Azure Portal. Click Active Directory, click the directory, choose the application, and click Configure.
 ** dynamics365.clientSecret - The application secret that you created in the app registration portal for your app. This should not be used in a native app, because a client secret cannot be reliably stored on a device. The client secret is required for web apps and web APIs, which have the ability to store the client secret securely on the server side.
 ** dynamics365.tokenRequestEndpoint - The token endpoint to call to get an access token. +
-Example: `+https://login.windows.net/TENANT_ID/oauth2/token` where TENANT_ID is the Azure Active Directory ID.
+Example: `+https://login.windows.net/TENANT_ID/oauth2/token+` where TENANT_ID is the Azure Active Directory ID.
```

Comment:
Should it be `TENANT_ID` ?

Reply:
TENANT_ID is a placeholder and not an actual value.